uberenv.py

```diff
@@ -284,6 +284,17 @@ def parse_args():
             opts["mirror"] = pabs(opts["mirror"])
     return opts, extras
 
+def have_internet(host="llnl.gov", port=80, timeout=3):
+    try:
+        s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        s.settimeout(3)
+        s.connect((host, port))
+        return True
+    except socket.error as ex:
+        print(ex)
```

Comment:
Should this print out the exception? It seems like the exception is expected on some platforms, so it'd be reasonable to omit the message.

Reply:
Fixed.